src/GuestConfiguration/GuestConfiguration/Properties/Resources.resx

```diff
@@ -142,7 +142,7 @@
     <value>Invalid report Id {0}.</value>
   </data>
   <data name="InvalidRGOrVMName" xml:space="preserve">
-    <value>Invalid resource group name {0} or VM name {1}. Please check whether the resource group and VM exist in subscription.</value>
+    <value>Invalid resource group name {0} or VM name {1}. Please check whether the resource group and VM exist in subscription. Use cmdlet Select-AzSubscription to select a subscription.</value>
```

Comment:
Use Select-AzContext to select a context with a different subscription

Reply:
done